### ISSUE TITLE
Entry SEO data being duplicated to other entries, fallbacks not working

### ISSUE THREAD

**Issue by joelstrom** [nystudio107/craft-seomatic#994]
A lot of entries on one of our large multi-site websites has started to output wrong SEO data. 
This seems to happen in one of two cases:

1. Entry does not have custom meta data filled in
2. Entry does not have data filled in fields that are defined in Content SEO

Expected behaviour is that if no custom meta data or Content SEO defined fields have been filled in the entry, then it would instead use the default settings defined in Content SEO. In case it can not use Content SEO data for some reason, it would fall back to Global SEO data.

Instead what is being shown in these cases is some SEO data from another entry completely. 
The problem seems to be happening in a structure section only, and the entry which SEO data is now being used across multiple entries is also residing in this same section.
This wrong SEO data is shown in both SEO previews as well as being generated on the frontend.

Needless to say it's a pretty huge issue for us at the moment!

- Plugin version: 3.4.15
- Craft version: 3.7.14

**Comment by khalwat** [comment]
> Expected behaviour is that if no custom meta data or Content SEO defined fields have been filled in the entry, then it would instead use the default settings defined in Content SEO. In case it can not use Content SEO data for some reason, it would fall back to Global SEO data.

Yes, this is exactly how it should be working.

If you are only having issues in one Section (a structure), does that mean it is working as expected in other sections?

Are you having this issue only on this one website? If so, what is different about websites where it is working as expected?

**Comment by khalwat** [comment]
It sounds to me like somehow there are fixed values stored in the Content SEO settings. Can you try going in and re-saving the Content SEO settings for this Section?

**Comment by joelstrom** [comment]
Re-saving the Content SEO settings seemed to have worked! I did have to go through all of the sites and save them separately.

To reply to your earlier message @khalwat:

- Yes the issue was only appearing in one structure section! We have 2 other structures that were not affected by this.
- So far we have only noticed the issue on this particular website
+ The buggy structure section was incredibly slow after recent Craft updates (loading the section was taking 20-60s, saving an entry could take the same amount of time, sometimes leading to timeouts) but that got fixed after we ran Craft's repair structure command and did some configs related to entry revisions (edit: the SEOmatic related bug was already appearing before running this command, we thought it could have fixed it but it did not)

I will keep an eye out for this and hope that the issue does not appear again. Case closed for now?
